src/parser/scanner.lex

```diff
@@ -543,7 +543,7 @@ RECOVER                     ([Rr][Ee][Cc][Oo][Vv][Ee][Rr])
                             }
 "#".*                       // Skip the annotation
 "//".*                      // Skip the annotation
-"--".*                      // Skip the annotation
+"-- ".*                     // Skip the annotation
```

Comment:
If there is no special reason, please drop this commit(#2115). Because it will not be supported any more in 2.0

Reply:
Can I Mark it and Push a pr to fix this after the all commits merged into cloud? Otherwise Testing is very time-comsuming